packages/components/src/components/ErrorBoundary/ErrorBoundaryScreen.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { FallbackProps } from "react-error-boundary";
+import { View, SafeAreaView } from "react-native";
+import { Button } from "../Button";
+import { EDSStyleSheet } from "../../styling";
+import { useStyles } from "../../hooks/useStyles";
+import { Typography } from "../Typography";
+
+export const ErrorBoundaryScreen = ({ resetErrorBoundary }: FallbackProps) => {
+    const styles = useStyles(theme);
+    return (
+        <SafeAreaView style={styles.safeAreaContainer}>
+            <View style={styles.container}>
+                <Typography color={styles.title_color.color} style={styles.text}>
+                    Error
+                </Typography>
+                <Typography style={styles.text}>
+                    Something unexpected happened, and the app crashed. You can restart the app
+                    below.
+                </Typography>
+                <Typography style={styles.text}>
+                    The error information has been forwarded to our team, and we will look into it.
```

Comment:
The phrasing of this text forces a user of the `ErrorBoundaryScreen` to implement tracking.
Should this be rephrased like a politician would so that we don't make the user think it is automatically handled?

Reply:
I'll ask Emilie about the phrasing

Reply:
![image](https://github.com/equinor/mad/assets/111871753/43a2cc1f-cd64-4c4e-8a84-c32fc2dbd03f)


Something like this?